techdocs/source/index.html.md.erb

```diff
@@ -7,7 +7,7 @@ weight: 10
 
 
 
-<%= warning_text("The project is paused, to get in contact whilst it's in this state, please email [Life Events Platform Feedback](life-events-platform-feedback@digital.cabinet-office.gov.uk).") %>
+<%= warning_text("The project is paused, to get in contact whilst it's in this state, please email life-events-platform-feedback@digital.cabinet-office.gov.uk.") %>
```

Comment:
Does this hyperlink? Or do you need `[life-events-platform-feedback@digital.cabinet-office.gov.uk](mailto:life-events-platform-feedback@digital.cabinet-office.gov.uk)` ?

Reply:
That doesn't hyperlink, but the warning text also doesn't render HTML